libs/angular/src/services/jslib-services.module.ts

```diff
@@ -629,6 +629,7 @@ const safeProviders: SafeProvider[] = [
       LOGOUT_CALLBACK,
       BillingAccountProfileStateService,
       TokenServiceAbstraction,
+      AuthService,
```

Comment:
This needs to have the `Abstraction` suffix.

```suggestion
      AuthServiceAbstraction,
```

Reply:
good catch, thank you